utils/build-apple-framework.sh

```diff
@@ -98,22 +99,15 @@ function create_universal_framework {
   cd ./destroot/Library/Frameworks || exit 1
 
   local platforms=("$@")
+  local args=""
 
   echo "Creating universal framework for platforms: ${platforms[*]}"
 
   for i in "${!platforms[@]}"; do
-    platforms[$i]="${platforms[$i]}/hermes.framework/hermes"
-  done
-
-  lipo -create -output "${platforms[0]}" "${platforms[@]}"
-
-  # Once all was linked into a single framework, clean destroot
-  # from unused frameworks
-  for platform in "${@:2}"; do
-    rm -r "$platform"
```

Comment:
Any particular reason why this has been removed? This should be reverted. We need to clean up `destroot` to only contain the artifacts that are used in production.

This is because CircleCI will package the entire `destroot` folder into a tarball that is distributed along the `npm` package. If we don't clean it up, we are going to ship one `xcframework` and every platform separately as well.

Reply:
OK, I've reverted that removal